lib/errors/errors.go

```diff
@@ -102,4 +102,8 @@ var (
 	InflationPFAmountMissMatched              = NewError(184, "inflation pf amount missmatched")
 	InflationPFFundingAddressMissMatched      = NewError(185, "inflation pf funding address missmatched")
 	CongressAddressMisMatched                 = NewError(186, "congress address mismatched")
+	FrozenAccountHasMoreThanOneOperation      = NewError(187, "Transaction from frozen account can have only one operation")
+	FrozenAccountCannotCreateAccount          = NewError(188, "Frozen account cannot create another account")
+	InflationPFOperationNotPermitted          = NewError(189, "Inflation PF operation is not permitted")
+	FrozenAccountMustWithdrawToOriginal       = NewError(190, "frozen account can only withdraw to the original account")
```

Comment:
We discussed that before. Why do you want to change it now ?

Reply:
To make it simple.
When we let frozen account do functions as ordinal account then,
we should think about all of the side effects.

Reply:
This has already been done. And it's not making it simpler, you are adding more code to support that restriction.